packages/quantic/force-app/main/default/lwc/quanticFoldedResultList/quanticFoldedResultList.js

```diff
@@ -134,7 +134,7 @@ export default class QuanticFoldedResultList extends LightningElement {
 
   registerTemplates() {
     this.dispatchEvent(
-      new CustomEvent('registerresulttemplates', {
+      new CustomEvent('quantic__registerresulttemplates', {
```

Comment:
search for `onregisterresulttemplates` to find places where this need to be updated